ush/parsing_namelists_MOM6.sh

```diff
@@ -60,32 +53,83 @@ fi
 
 echo "$(cat input.nml)"
 
+# Source functions from this file for filling in templates
+source "${HOMEgfs}/ush/atparse.bash"
 
-#Copy MOM_input and edit:
-${NCP} -pf "${HOMEgfs}/parm/ufs/mom6/MOM_input_template_${OCNRES}" "${DATA}/INPUT/"
-sed -e "s/@\[DT_THERM_MOM6\]/${DT_THERM_MOM6}/g" \
-    -e "s/@\[DT_DYNAM_MOM6\]/${DT_DYNAM_MOM6}/g" \
-    -e "s/@\[MOM6_RIVER_RUNOFF\]/${MOM6_RIVER_RUNOFF}/g" \
-    -e "s/@\[MOM6_THERMO_SPAN\]/${MOM6_THERMO_SPAN}/g" \
-    -e "s/@\[MOM6_USE_LI2016\]/${MOM6_USE_LI2016}/g" \
-    -e "s/@\[MOM6_USE_WAVES\]/${MOM6_USE_WAVES}/g" \
-    -e "s/@\[MOM6_ALLOW_LANDMASK_CHANGES\]/${MOM6_ALLOW_LANDMASK_CHANGES}/g" \
-    -e "s/@\[NX_GLB\]/${NX_GLB}/g" \
-    -e "s/@\[NY_GLB\]/${NY_GLB}/g" \
-    -e "s/@\[CHLCLIM\]/${CHLCLIM}/g" \
-    -e "s/@\[DO_OCN_SPPT\]/${OCN_SPPT}/g" \
-    -e "s/@\[PERT_EPBL\]/${PERT_EPBL}/g" \
-    -e "s/@\[MOM6_DIAG_COORD_DEF_Z_FILE\]/${MOM6_DIAG_COORD_DEF_Z_FILE}/g" \
-    -e "s/@\[TOPOEDITS\]/${TOPOEDITS}/g" \
-    -e "s/@\[MOM6_DIAG_MISVAL\]/${MOM6_DIAG_MISVAL}/g" \
-    -e "s/@\[ODA_INCUPD_NHOURS\]/${ODA_INCUPD_NHOURS}/g" \
-    -e "s/@\[ODA_INCUPD\]/${ODA_INCUPD}/g" "${DATA}/INPUT/MOM_input_template_${OCNRES}" > "${DATA}/INPUT/MOM_input"
-rm "${DATA}/INPUT/MOM_input_template_${OCNRES}"
+# ================================================================
+# MOM_input
```

Comment:
A lot of commented-out code in this one. Can you clean it up?

Reply:
done.